index.html

```diff
@@ -215,6 +214,14 @@ <h1 class="title">
                                         <a href="https://github.com/MozillaTN">GitHub</a>
                                         <a href="https://twitter.com/mozillaTN">Twitter</a>
                                     </div>
+                                    <div class="box">
+                                        <p class="subtitle">vMozilla</p>
```

Comment:
I think it's better to name regional groups after cities/states so that others can recognize what region it is

Reply:
I think so he has added Mozilla community for Vellore Institute of Technology, Vellore, so I don't think that it can be named as a regional group.

Reply:
From Telegram discussions, it seems that Mozilla Community for VIT itself is the regional group(pardon me if I misunderstood) which again raises the question that should it come under regional groups, if so then should it be named after the place rather than the group?

Reply:
Yep, after discussing over telegram, I've realized its in everyone's best interest to call the umbrella Community as `vMozilla` :1st_place_medal: 

The Campus Clubs are therefore shortened to `vMozilla <Geolocation>` :2nd_place_medal: 

-----

So, I'll have a placeholder commented out for future (upcoming) CCs at other @VITUni campuses.

## TL;DR : All communities referencing @VITUni <
# New names are shorter, meaningful and #include 'Mozilla' too!
::|Name|Purpose|Notes
----|----|----|----
RC-1|vMozilla|Umbrella Community|Incl. alumni/staff/enrolled/associates to @VITUni)
CC-1|vMozilla, Vellore|Campus Club|Incl. enrolled volunteers to @VITUni-#Vellore)
CC-2|vMozilla, Chennai|Campus Club|Incl. enrolled volunteers to @VITUni-#Chennai)
CC-3|vMozilla, Amrawati|Campus Club|Incl. enrolled volunteers to @VITUni-#Amrawati)
CC-4|vMozilla, Bhopal|Campus Club|Incl. enrolled volunteers to @VITUni-#Bhopal)
CC-5|vMozilla, Jaipur|Campus Club|Incl. enrolled volunteers to @VITUni-#Jaipur)

Reply:
Resolves : :+1: 

- [x] Naming of Campus Club (@VITUni-<GeøŁøcation>)